docs/components/sidebar-nav.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sidebar with Navigation
```

Comment:
Not sure if the url (which is the name of the file) should be spelled out like in the title of the page.